src/FeatureExtractor/Mobilenet.js

```diff
@@ -243,7 +243,7 @@ class Mobilenet {
     await tf.nextFrame();
     this.isPredicting = true;
     const predictedClass = tf.tidy(() => {
-      const processedImg = imgToTensor(imgToPredict);
+      const processedImg = imgToTensor(imgToPredict, [224, 224]);
```

Comment:
This and all the next methods should use the `IMAGESIZE` [(line 16)](https://github.com/ml5js/ml5-library/blob/cb8651e477d0f307ecf8bc7a3f6e650ac78a3c2e/src/FeatureExtractor/Mobilenet.js#L16) variable instead of hard coding the resize values.